### ISSUE TITLE
ios Version 14.2.0/de_DE sends 'location' as _type on 'steps' message - HTTP API

### ISSUE THREAD

**Issue by balu-** [owntracks/ios#674]
Hi there, 

I'm using the new version 14.2.0/de_DE and noticed that the version seems to send the value 'location' as '_type' for a message that contains step data. 
In previous versions the _type value was set to 'steps' for this kind of messages,
are there changes i'm missing?

Sample Data
```
{"from":1617819838,"distance":0,"steps":0,"_type":"location","tst":1617835016,"topic":"owntracks\/owntracks\/<deviceID>\/step","floorsdown":0,"to":1617821638,"floorsup":0}
```

**Comment by ckrey** [comment]
Well spotted, this regression bug was introduced with f35544b5d242b3b998dd35c479b2ba7aa2e294c1

**Comment by ckrey** [comment]
Released in 14.2.2
